lib/rules/lines-between-class-members.js

```diff
@@ -55,6 +93,7 @@ module.exports = {
         options[0] = context.options[0] || "always";
         options[1] = context.options[1] || { exceptAfterSingleLine: false };
 
+        const configureList = typeof options[0] === "object" && options[0].enforce ? options[0].enforce : [];
```

Comment:
I think that if `options[0]` is a string, assigning `[{ blankLine: options[0], prev: "*", next: "*" }]`, can remove the L261 statement.
What do you think?

Reply:
@ota-meshi yes, I've refactored the code.